src/aggUtils.js

```diff
@@ -1,4 +1,21 @@
 let _ = require('lodash/fp')
+let isSubset = (a, b) =>
+  !_.flow(
+    _.difference(a),
+    _.size
+  )(b)
+let metrics = [
+  'avg',
+  'max',
+  'min',
+  'sum',
+  'value_count',
+  'weighted_avg',
+  'cardinality',
+  'percentiles',
```

Comment:
This won't work because `percentiles` does not have a `value` property on the response, it has `values`. You should check the es docs on all of these to be sure